### PR TITLE
Orderimportdraft inventorymode 1291

### PR DESCRIPTION
**Pull request by katmatt**

### CODE REVIEW

File: commercetools-models/src/test/java/io/sphere/sdk/attributestutorial/ProductTypeCreationDemoIntegrationTest.java [hunk 1]
```diff
@@ -528,6 +529,7 @@ public void orderImportExample() throws Exception {
                 .build();
         final OrderImportDraft orderImportDraft = OrderImportDraftBuilder
                 .ofLineItems(EURO_20, OrderState.COMPLETE, asList(lineItemImportDraft))
+                .inventoryMode(InventoryMode.NONE)
```
Comment (lauraluiz) [inline commercetools-models/src/test/java/io/sphere/sdk/attributestutorial/ProductTypeCreationDemoIntegrationTest.java]: This is the test for product types, I would rather remove this from here and add a test in `OrderImportCommandIntegrationTest` that checks the default behaviour (NONE) when you set nothing and trying to set one in particular.

**Comment by lauraluiz** [comment]
Besides the test, the rest looks good 👍

File: commercetools-models/src/test/java/io/sphere/sdk/orders/commands/OrderImportCommandIntegrationTest.java [hunk 2]
```diff
@@ -350,6 +349,19 @@ public void customerId() throws Exception {
...
+    public void inventoryMode() throws Exception {
+        testOrderAspect(builder -> {
+                    builder.inventoryMode(null);
+                },
+                order -> assertThat(order.getInventoryMode()).isEqualTo(InventoryMode.NONE));
+
+        testOrderAspect(builder -> {
+                    builder.inventoryMode(InventoryMode.NONE);
+                },
+                order -> assertThat(order.getInventoryMode()).isEqualTo(InventoryMode.NONE));
```
Comment (lauraluiz) [inline commercetools-models/src/test/java/io/sphere/sdk/orders/commands/OrderImportCommandIntegrationTest.java]: This could be true even if we forgot to send the inventory mode with the builder. Using e.g `InventoryMode.RESERVE_ON_ORDER` will make a better assertion ;)

**Comment by lauraluiz** [comment]
closes https://github.com/commercetools/commercetools-jvm-sdk/pull/1304

**Comment by lauraluiz** [comment]
closes #1291
